session_ii/javascript-fundamentals/session1/homework/mpricu/array.js

```diff
@@ -0,0 +1,39 @@
+let array1 = [
+    {
+        x: "Forever",
+
+        getX() {
+            return this.x;
+        }
+    }, 
+    {
+        x: "Young",
+
+        getX() {
+            return this.x;
+        }
+    },
+    {
+        x: "Big",
+
+        getX() {
+            return this.x;
+        }
+    },
+    {
+        x: "in",
+
+        getX() {
+            return this.x;
+        }
+    },
+    {
+        x: "Japan",
+
+        getX() {
+            return this.x;
+        }
+    },
+];
+
+export {array1} ;
```

Comment:
This works. Note that export can be combined with let, const and var in a single statement.